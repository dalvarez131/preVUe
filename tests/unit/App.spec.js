import { shallowMount, mount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App.vue', () => {
  it('renders correct', () => {
    const wrapper = shallowMount(App);
    expect(wrapper.element).toMatchSnapshot();
  }),
    it('contains a section that wraps router-view', () => {
      const wrapper = shallowMount(App, {});
      expect(wrapper.contains('section')).toBe(true);
    });
});
