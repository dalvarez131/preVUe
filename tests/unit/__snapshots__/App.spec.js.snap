// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders correct 1`] = `
<div
  class="hero is-fullHeight"
  id="app"
>
  <navbar-stub />
   
  <section
    class="section"
  >
    <router-view />
  </section>
</div>
`;
